//package com.sincetimes.redwar.game.comnunicate.protocol;
message RceQueryReceiveWishItem
{
	optional string planetId = 1;
	optional string itemSku = 2;
	optional string id = 3;
}